docs/visual-basic/getting-started/whats-new.md

```diff
@@ -64,6 +64,10 @@ For example, the following method call has two positional arguments between a na
 StudentInfo.Display("Mary", age:=19, #9/21/1998#)
 ```
 
+[`Private Protected` member access modifier](../language-reference/modifiers/private-protected.md)
+
+This new keyword combination defines a member that is accessible by all members in its containing class as well as by types derived from the containing class, but only if they are found in the containing assembly. Because structures cannot be inherited, `Private Protected` can only be applied to the members of a class.
```

Comment:
consider "also found in the containing assembly" 

Otherwise fantastic! Let's post it.